scripts/flow/runFlow.js

```diff
@@ -21,11 +22,21 @@ async function runFlow(renderer, args) {
     if (process.platform === 'win32') {
       cmd = cmd.replace(/\//g, '\\') + '.cmd';
     }
+
+    // Symlink render flowconfig file to the root of the project so that it
+    // works with editor integrations. This means that the Flow config used by
+    // the editor will correspond to the last renderer you checked.
```

Comment:
Turns out Flow doesn't detect changes to the symlink so I'll need to be a bit more clever

Reply:
Ended up just copying the file instead